src/assets/theme/theme.ts

```diff
@@ -127,6 +127,19 @@ const theme: Theme = {
     '--label-color': 'var(--color-black-90)',
     '--label-color-disabled': 'var(--color-black-40)',
   },
+  collapsible: {
+    '--heading-background-color': 'var(--color-white)',
+    '--heading-border-color': 'var(--color-bus)',
+    '--heading-color': 'var(--color-bus)',
+    '--heading-background-color-expanded': 'var(--color-bus)',
+    '--heading-border-color-expanded': 'var(--color-bus)',
+    '--heading-color-expanded': 'var(--color-white)',
+    '--heading-background-color-expanded-hover': 'var(--color-bus-dark)',
+    '--heading-border-color-expanded-hover': 'var(--color-bus-dark)',
+    '--heading-color-expanded-hover': 'var(--color-white)',
+    '--heading-background-color-hover': 'var(--color-bus-light)',
+    '--heading-color-hover': 'var(--color-bus-dark)',
+  },
```

Comment:
btw do these variables override the ones in collapsible.module.scss?

Reply:
Yes they do. The idea is to have the all theme css variables in one single file